app/views/catalog/_show_sections/_default_contents.html.erb

```diff
@@ -43,7 +35,7 @@
                         <% child.xpath('file').each do |file| %>
                           <li class="file">
                             <span class="label">File</span>
-                            <% if can_manage %>
+                            <% if can? :manage_content, object %>
```

Comment:
Is there anything relevant from that discussion we should add here (for posterity or context for future developers)?